src/main/java/jdwp/Translator.java

```diff
@@ -258,6 +362,36 @@ public static LocationImpl locationLookup(String func, int line) {
 	}
 
 
+	public static void translateReferenceTypes(Map<Long, ReferenceType> referenceTypes, MiSymbolInfoFunctionsInfo response) {
+		Map<String, ReferenceType> types = new HashMap<>();
+		for(SymbolFileInfo symbolFile : response.getSymbolFiles()) {
+			for(Symbols symbol : symbolFile.getSymbols()) {
+				var index = symbol.getName().indexOf("::");
+				if (index != (-1)) {
```

Comment:
@jeffmaury What do you think about instead of generating the full output for `-symbol-info-functions` and filtering out with an `if` check here, we can call the GDB MI method at [loadSymbols()](https://github.com/nativejdb/nativejdb/pull/71/files#diff-818cd7a448edb115cc7340441d8bf5a1d33760422cb675a2058b7073f0a72f3bR113) with `createMiSymbolInfoFunctions("", "::", 0, false);`?

Since the output is huge, it takes a long time to generate and respond...